minesweeper.py

```diff
@@ -320,9 +362,15 @@ def new_game(event=None):
 if root.tk.call("tk", "windowingsystem") == "aqua":
     top_menu.add_cascade(label="Game", menu=top_menu_game)
     top_menu_game.add_command(label="New Game", accelerator="F2", command=new_game)
+    top_menu_game.add_command(
+        label="High Scores", accelerator="F6", command=create_highscores_window
+    )
     top_menu_game.add_command(label="Quit Game", accelerator="F10", command=quit_game)
 else:
     top_menu.add_command(label="New Game", accelerator="F2", command=new_game)
+    top_menu_game.add_command(
```

Comment:
```suggestion
    top_menu.add_command(
```

This makes the menu actually appear on my system.